### PR TITLE
Ensure SBOM FreeType version is set correctly for system and bundled settings

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Fixes https://github.com/adoptium/temurin-build/issues/3493

Determines what type of FreeType build it was "system" or "bundled", then look for the freetype.h from that freetype installation/openjdk bundle, to determine the FREETYPE_MAJOR,MINOR,PATCH

### CODE REVIEW

**Comment by andrew-m-leonard** [comment]
> Will this still pick up things properly in the case where we are pulling a version from git? Other than checking that (and a personal hatred of using bash arrays) this LGTM (Not looked in explicit detail, but I trust that you've tested it, and I really want this in before the October release cycle!)

@sxa Yes, works for all.. the FREETYPE_CFLAGS gets pointed at by autoconf to either the cloned freetype includes, or the system installed one...

File: sbin/build.sh [hunk 1]
```diff
@@ -920,6 +920,62 @@ checkingToolSummary() {
    sed -n '/^Tools summary:$/,$p' "${inputConfigFile}" > "${outputConfigFile}"
 }
 
+# Determine FreeType version being used in the build from either the system or bundled freetype.h definition
+addFreeTypeVersionInfo() {
+   # Default to "system" the jdk8 only value
+   local FREETYPE_TO_USE="system"
+   if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" != "${JDK8_CORE_VERSION}" ]; then
```
Comment (karianna) [inline sbin/build.sh:927]: should that be `==`? Seems different to the comment above

**Review by karianna — CHANGES_REQUESTED** [review]

File: sbin/build.sh (hunk 1 above)
Comment (andrew-m-leonard) [reply sbin/build.sh:927]: != is correct, the comment is missleading, i've changed it
jdk8 can only be system
jdk-11+ supports bundled or system

**Review by karianna — APPROVED** [review]

**Review by sxa — APPROVED** [review]
Based on my earlier question and this:

> != is correct, the comment is missleading, i've changed it jdk8 can only be system jdk-11+ supports bundled or system

do we have builds  where we are building freetype from a downloaded source? Approving regardless ...

**Comment by andrew-m-leonard** [comment]
> Based on my earlier question and this:
> 
> > != is correct, the comment is missleading, i've changed it jdk8 can only be system jdk-11+ supports bundled or system
> 
> do we have builds where we are building freetype from a downloaded source? Approving regardless ...

@sxa Yes jdk8u, Windows & Mac

**Pull request merged**